.vscode/settings.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I know this is kinda piggybacking, but I made these changes to make editing these files easier and decided to just commit them. If people think it should be split out or removed, I'm happy to make that change.

Reply:
Looks like this file has since been removed; resolving.